components/channel_header/channel_header.js

```diff
@@ -438,40 +438,49 @@ class ChannelHeader extends React.PureComponent {
             />
         );
 
+        let membersIconClass = 'member-rhs__trigger channel-header__icon channel-header__icon--left channel-header__icon--wide';
```

Comment:
Would you mind moving this all into the `if (!isDirect)` block below? Since it's only needed in there, we may as well keep it a bit more contained

Reply:
✔️